### DISCUSSION TITLE
Canva Pro with GitHub Education.

### DISCUSSION THREAD

**Discussion by Thegameofdead** [community/community#54112] [GitHub Education]
### Hi everyone,

Hello, dudes.
A year ago I had Canva Pro through GitHub Education but as you know, it only lasts 1 year. Now I want to have it again but I don't see anything about Canva Pro, does anyone know where I can get Canva Pro for free with GutHub Education as before?

**Comment by samus-aran** [comment]
Hi @Thegameofdead  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers).

**Reply by Designbyasma** [reply]
Yes

**Comment by Designbyasma** [comment]
Yes I ' need canva pro

**Comment by CanvaProPricing** [comment]
[Canva Pro Pricing](https://github.com/CanvaProPricing/Cost/): Just $8/Month! https://github.com/CanvaProPricing/Cost/

**Comment by RiddhiMFM** [comment]
Hii ..!! How can i apply to your students developer pack to get access to canva pro from students account.

**Comment by alsami16** [comment]
I need canva pro or any other alternative like canva
